crates/wasm-shrink/src/main.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::Result;
+use structopt::StructOpt;
+use wasm_shrink::WasmShrink;
+
+fn main() -> Result<()> {
```

Comment:
Could this be located at `src/bin/wasm-shrink.rs` with the rest of our binaries?

I was looking at https://github.com/WebAssembly/wasp the other day as well and noticed how the CLI organization there was one tool with subcommands which personally I feel like maps a lot better to what this suite of `wasm-tools` we have in our repository are. One day I'd like to move to that model to reduce duplication between all the tools ideally, but doesn't necessarily have to happen here I think.

Reply:
I wanted to be able to do `cargo install wasm-shrink`, but yeah if we make `wasm-tools {shrink,smith,wasm2wat,etc...}` I would be happy with that.